changelogs/fragments/1413-proxmox-features.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - proxmox - add features option to LXC (https://github.com/ansible-collections/community.general/pull/1413).
```

https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment
> Each changelog entry must contain a link to its issue between parentheses at the end. If there is no corresponding issue, the entry must contain a link to the PR itself.
```suggestion
  - proxmox - add features option to LXC (https://github.com/ansible-collections/community.general/issues/816).
```